starters/next-drupal-starter/pages/pages/index.js

```diff
@@ -23,7 +26,12 @@ export default function PagesList({ hrefLang, pages }) {
               <li className="prose justify-items-start" key={id}>
                 <h2>{title}</h2>
                 <div dangerouslySetInnerHTML={{ __html: body.summary }} />
-                <Link passHref href={`/pages${path.alias}`}>
+                <Link
```

Comment:
@CobyPear on my last run through, happened to notice that the 'Read more' link is to /undefined/about-umami. It looks to be working as expected on the newest canary.

Aside from that, the rest of this looks good to me.

Reply:
@backlineint I see the alias for `articles` includes `/articles` in it, but `pages` does not include `/pages`. Is this expected or an inconsistency on the backend?